common/src/main/java/com/wynntils/features/trademarket/TradeMarketPriceConversionFeature.java

```diff
@@ -45,13 +44,4 @@ public void onClientChat(ChatSentEvent event) {
             McUtils.sendChat(price);
         }
     }
-
-    @SubscribeEvent
-    public void onGuiOpen(ScreenOpenedEvent.Post event) {
-        if (TRADE_MARKET_PATTERN
-                .matcher(event.getScreen().getTitle().getString())
-                .matches()) {
-            shouldConvert = false;
-        }
-    }
```

Comment:
You are no longer able to open the tm until inputting something so I presume this is no longer needed